frontend/src/metabase/components/LoadingAndErrorWrapper/LoadingAndErrorWrapper.tsx

```diff
@@ -0,0 +1,137 @@
+import { useInterval } from "@mantine/hooks";
+import cx from "classnames";
+import {
+  type CSSProperties,
+  Children,
+  type ReactNode,
+  useEffect,
+  useState,
+} from "react";
+import { t } from "ttag";
+
+import LoadingSpinner from "metabase/components/LoadingSpinner";
+import CS from "metabase/css/core/index.css";
+
+interface LoadingAndErrorWrapperProps {
+  loading?: boolean;
+  error?: any;
```

Comment:
not sure if using `unknown` might make more sense for errors, since you could then cast it to any error types